Automattic/wp-calypso
# Launchpad: Add the task redirect action to the navigator

Related to #

## Proposed Changes

* Adds the redirect behavior to the tasks on the Navigator. Prior to this change, clicking on the tasks would have no effect. This PR handles the click action by using the `setUpActionsForTasks` function from the `@automattic/launchpad` package.
* Added a new component to wrap the Launchpad around and avoid repetition with the task setup.
* Currently, we are not handling the tasks that fire modals. I will work on that in a followup PR.


## Testing Instructions



* Use the Calypso live link below
* Go to the Customer Home on a site with the Launchpad
* Add the flag to the URL: `flags=launchpad/navigator`
* Click on the Ring in the master bar, which should show the Navigator modal
* Click on some tasks and make sure the user is redirected
* Ensure the track events are fired with the context `navigator`

## Pre-merge Checklist
